### PR TITLE
Update README file

### PR DESCRIPTION
**Pull request by trickeydan**

### CODE REVIEW

**Comment by 8BitJosh** [comment]
I am ok with the changes in the README, are we going to remove the setup.sh script from the repo as all these steps are now contained in the README

**Comment by trickeydan** [comment]
> I am ok with the changes in the README, are we going to remove the setup.sh script from the repo as all these steps are now contained in the README

I'm happy to remove it if you think that would be appropriate? The script doesn't work in my environment anyway (ARM VM) so I just wanted to make sure that I documentes multilib wasn't needed.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+```shell
+sudo apt install gcc-multilib
+```
+
+Once your have installed the system dependencies, you will need to setup the repository:
+
+```shell
+git clone git@github.com:srobo/robot-image.git
+git submodule update --init --recursive
```
Comment (8BitJosh) [inline README.md:29]: You need to cd into the directory before running the submodule command

Comment (raccube) [reply README.md:29]: Better yet `git clone --recursive`

**Review by raccube — CHANGES_REQUESTED** [review]

**Review by raccube — APPROVED** [review]

**Pull request merged**